.kokoro

```diff
@@ -49,6 +49,30 @@ fix_bazel_imports() {
   trap reset_imports EXIT
 }
 
+run_bazel() {
+  fix_bazel_imports
+
+  ./.kokoro-ios-runner/bazel.sh test //components/... 8.2.0
+}
+
+run_cocoapods() {
+  gem install xcpretty cocoapods --no-rdoc --no-ri --no-document --quiet
+  pod --version
+
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    # Move into our cloned repo
+    cd github/repo
+  fi
+
+  export DEVELOPER_DIR=/Applications/Xcode_8.2.app/Contents/Developer
```

Comment:
We should have a comment explaining that this is older than our supported Xcode and that we plan on changing it when 8.3.3 becomes available

Reply:
Should I rename my Xcode installations to follow this pattern?  This is mine:

```bash
 ls -1 /Applications/ | grep Xcode
Xcode.app
Xcode8.3.3.app
```

Reply:
Maybe as a future improvement, just loop over all installed Xcode versions?

```bash
$ mdfind "kMDItemCFBundleIdentifier == 'com.apple.dt.Xcode'"
/Applications/Xcode.app
/Applications/Xcode8.3.3.app
```

Reply:
Done.